sql/testdata/family

```diff
@@ -130,3 +130,25 @@ query III
 SELECT * FROM unsorted_colids
 ----
 1 2 3
+
+# Check that family bookkeeping correctly tracks column renames
+statement ok
+CREATE TABLE rename_col (a INT PRIMARY KEY, b INT, c STRING, FAMILY (a, b), FAMILY (c))
+
+statement ok
+ALTER TABLE rename_col RENAME b TO d
+
+statement ok
+ALTER TABLE rename_col RENAME c TO e
+
+query TT
+SHOW CREATE TABLE rename_col
+----
+rename_col  CREATE TABLE rename_col (
```

Comment:
double space? also, how do you feel about this indentation:
rename_col CREATE TABLE rename_col (
  a INT NOT NULL,
  ....
)
```
?